content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst

```diff
@@ -0,0 +1,51 @@
+===================================
+EU intra-community distance selling
+===================================
+
+**Distance selling within the European Union** implies that organizations that sell goods and
+services cross-border to a private consumer (B2C) in an EU Member State must ensure that the VAT is
+paid to the Member State in which the goods or services are delivered.
+
+.. note::
+   This is also valid if your organization is located outside of the European Union.
+
+While this regulation mainly applies to :doc:`eCommerce <../../../../websites/ecommerce>` sales to
+private EU consumers, it is also valid for regular cross-border sales.
+
+The Union **One-Stop Shop (OSS)** is an online portal that businesses can use to register for the
+OSS and declare their transactions.
+
+The **EU intra-community distance selling** feature helps comply with this regulation by creating
+and configuring new :doc:`fiscal positions <fiscal_positions>` and :doc:`taxes <taxes>` based on
+your company's country.
+
+.. seealso::
+   - `European Commission: OSS | Taxation and Customs Union
+     <https://ec.europa.eu/taxation_customs/business/vat/oss_en>`_
+
+.. _eu_distance_selling/configuration:
+
+Configuration
+=============
+
+Go to :menuselection:`Accounting/Invoicing --> Settings --> Taxes`, then enable **EU intra-community
+distance selling** (or **EU Digital Goods VAT** if you created your database before July 1, 2021),
+and *Save*.
+
+.. important::
+   Please :ref:`upgrade the module <general/upgrade>` `l10n_eu_service` if you already installed it
+   before **July 1, 2021**, or if you activated the feature **EU Digital Goods VAT** in the
+   Accounting settings.
```

Comment:
Perhaps also a short mention that it means you need to update your Odoo instance it's source code if you are on premise too? I think quite some people wouldn't know that. You can link to the original docs about updates/deployments.

Reply:
Thanks for the feedback @Yenthe666 

I'll probably add this in the "general" doc, in the "upgrade" section referenced here rather than explain it in this documentation. This way, we don't overload all docs that require an upgrade, but if people need more information, they'll find it in the "upgrade apps and modules" section.

Since it's a change in 13.0, I'll do it with another PR.

Reply:
Sounds good, thanks :)